packages/backend/src/models/User.ts

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable no-shadow */
+import bcrypt from 'bcrypt';
+import mongoose from 'mongoose';
+
+export type UserDocument = mongoose.Document & {
```

Comment:
Any way to avoid duplicating code here?

Reply:
What code are you referring to? The UserDocument?